scripts/webpack/webpack.config.stats.ts

```diff
@@ -55,13 +58,21 @@ const makeConfig = (srcPath: string, name: string): webpack.Configuration => ({
     react: 'react',
     'react-dom': 'reactDOM',
   },
+  optimization: {
+    concatenateModules: false,
+  },
   plugins: [
     new CleanWebpackPlugin([paths.base('stats')], {
       root: paths.base(),
       verbose: false, // do not log
     }),
     new IgnoreNotFoundExportPlugin(),
     new webpack.DefinePlugin(config.compiler_globals),
+    // new BundleAnalyzerPlugin({
```

Comment:
It will be really useful to store reports as artifacts

Reply:
Yup, I plan to do that in PR gate work